installer/conf/logrotate.conf

```diff
@@ -6,3 +6,12 @@
     size 100M
     copytruncate
 }
+
+/var/opt/microsoft/omsagent/log/update_run_progress.log {
```

Comment:
What is the status multi-homing? If it has been enabled, then our runbook needs to be update to aware workspace ID as well. Who can confirm this?